testing/mock/test_system.cpp

```diff
@@ -270,12 +273,12 @@ void test_system::remove_sysfs() {
     }
     if (S_ISDIR(st.st_mode)){
       auto cmd = "rm -rf " + root_;
-      std::system(cmd.c_str());
+      result = std::system(cmd.c_str());
```

Comment:
We should probably wrap with a check like:
```C++
auto retval = std::system(cmd.c_str());
if (retval != 0) {
  if (retval == -1) {
    std::cerr << "Failed to fork process\n";
  }
  std::string msg = "Error running command " + cmd;
  msg += ": exit code is " + std::to_string(retval);
  std::cerr << msg << "\n";
}
```

Because this is in the cleanup phase, we probably don't care as much and so we shouldn't throw an exception.